testsuite/constant_pH.py

```diff
@@ -88,7 +88,8 @@ def test_ideal_titration_curve(self):
         RE = ReactionEnsembleTest.RE
         """ chemical warmup in order to get to chemical equilibrium before starting to calculate the observable "degree of association" """
         for i in range(40 * N0):
-            r = RE.reaction_constant_pH()
+            #r = RE.reaction_constant_pH()
+            r = RE.reaction(3)
```

Comment:
In the present branch this won't call the constant pH method. Only in the upcoming branch jonaslandsgesell:test_wang_landau_reaction_ensemble it will call the constant_pH method (iff you created a constant ph object)